### DISCUSSION TITLE
有没有大佬给个建议

### DISCUSSION THREAD

**Discussion by wtnl551** [share121/inter-knot#2646] [其他]
![微信图片_20240818095624](https://github.com/user-attachments/assets/6adcde9f-ec35-407c-b714-98b01c1d79f0)
混沌十二打不满

**Comment by wtnl551** [comment]
有专武的有克拉拉、老杨、流萤、符玄、花火

**Comment by Rolesican** [comment]
有两队了啊，可能是遗器问题，或者排轴手法问题，可以去b站上找专业up看看

**Reply by wtnl551** [reply]
知道了，谢谢大佬

**Comment by FWJ321** [comment]
为什么不练佩拉练瓦尔特

**Reply by wtnl551** [reply]
新手池出的瓦尔特，当时想着五星比四星好用，后来出了银狼又自己组了个配队。到后期打不过才把瓦尔特换下来但当时瓦尔特已经练满了，现在又没多的体力去养一点没练的角色
